apps/dapp-console-api/src/constants/envVars.ts

```diff
@@ -72,6 +72,18 @@ const envVarSchema = z.object({
   CB_VERIFICATION_ATTESTER: z
     .string()
     .default('0x357458739F90461b99789350868CD7CF330Dd7EE'),
+  FAUCET_CONTRACT_ADDRESS: z.string(),
```

Comment:
nit: make this an `Address` type: `FAUCET_CONTRACT_ADDRESS: z.custom<Address>()`